workspaces/core/src/select/Select.tsx

```diff
@@ -0,0 +1,150 @@
+import React, {forwardRef, useState} from "react";
+import ChevronDown from "../private/icons/ChevronDown";
+import ChevronUp from "../private/icons/ChevronUp";
+import Clear from "../private/icons/Clear";
+import {determineMenuBackgroundColor} from "./private/getBackgroundColor";
+import {getWrapperClasses} from "./private/getWrapperClasses";
+import Option from "./private/types/OptionProps";
+import SelectProps from "./private/types/SelectProps";
+import useEventListeners from "./private/useEventListeners";
+
+const Select = forwardRef<HTMLDivElement, SelectProps>(({
+  options,
+  label,
+  value,
+  disabled,
+  isError,
+  placeholderSlot,
+  placeholderValue,
```

Comment:
what is the difference between these two props?

Reply:
They are basically the same, but I saw that they were used previously in Select for the same thing, somewhere it was `placeholderSlot` and somewhere `placeholderValue` so I wanted to provide the same interface for this component for the sake of backwards compatibility